projects/spectator/jest/test/form-input/form-input.component.spec.ts

```diff
@@ -4,53 +4,51 @@ import { createHostFactory, SpectatorHost, Spectator, createComponentFactory } f
 import { FormInputComponent } from '../../../test/form-input/form-input.component';
 
 describe('FormInputComponent', () => {
-  let host: SpectatorHost<FormInputComponent>;
-  const createHost = createHostFactory({
```

Comment:
I'm not sure what these changes are.

Reply:
https://github.com/ngneat/spectator/pull/465#discussion_r684246251